lms/djangoapps/instructor/views/api.py

```diff
@@ -2327,6 +2327,32 @@ def list_financial_report_downloads(_request, course_id):
     return JsonResponse(response_payload)
 
 
+# Is the non_atomic_requests header correct?
```

Comment:
Cool. This was left by the previous author, but it seems like I should just remove the comment, then?